deploy_js_lambda.sh

```diff
@@ -0,0 +1,32 @@
+#/bin/bash
+
+STAGE=$1
+BUCKET=$2
```

Comment:
If we hard code the bucket name we expose it in the repo, i'm not comfortable making the name public (although the permissions are sufficient to prevent tampering). It also is another Guardian specific thing that consumers of the project would have to work around.